lib/okta.go

```diff
@@ -536,3 +536,39 @@ func (p *OktaProvider) Retrieve() (sts.Credentials, string, error) {
 
 	return creds, oktaCreds.Username, err
 }
+
+func (p *OktaProvider) GetSamlLoginURL() (*url.URL, error) {
```

Comment:
We've already broken this unholy seal elsewhere, but how do you feel about capitalizing `Saml --> SAML`? For case consistency.

Reply:
Good call!